src/containers/Feed/feeds.jsx

```diff
@@ -47,13 +48,38 @@ class Feed extends Component {
     this.setState({ monthCount: months });
   }
 
+  handleDelete = id => {
+    const { data, monthCount } = this.state;
+    message.warning('This Journal is deleted');
+    // 1- this card will be deleted from firbase store.
+    // 2- also it will be deleted from state as follows :
+    const deletedCardMonth = moment(data[0].timestamp).format('MMMM');
+    monthCount.map(month => {
```

Comment:
Maybe you could write this better to avoid the month.month down there 
```suggestion
    monthCount.map({ month } => {
```